semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -186,6 +189,61 @@ def to_markdown_attr(
             )
         )
 
+    def to_markdown_attribute_table(
+        self, semconv: BaseSemanticConvention, output: io.StringIO
+    ):
+        attr_to_print = []
+        for attr in sorted(
+            semconv.attributes, key=lambda a: "" if a.ref is None else a.ref
+        ):
+            if self.render_ctx.group_key is not None:
+                if attr.tag == self.render_ctx.group_key:
+                    attr_to_print.append(attr)
+                continue
+            if self.render_ctx.is_full or attr.is_local:
+                attr_to_print.append(attr)
+        if self.render_ctx.group_key is not None and not attr_to_print:
+            raise ValueError(
+                "No attributes retained for '{}' filtering by '{}'".format(
+                    semconv.semconv_id, self.render_ctx.group_key
+                )
+            )
+        if attr_to_print:
+            output.write(MarkdownRenderer.table_headers)
+            for attr in attr_to_print:
+                self.to_markdown_attr(attr, output)
+        attr_sampling_relevant = [
+            attr for attr in attr_to_print if attr.sampling_relevant
+        ]
+        self.to_creation_time_attributes(attr_sampling_relevant, output)
```

Comment:
This will change the output by reordering the paragraphs I think.

Reply:
Yeah, I think it doesn't matter too much.